commands/loot.js

```diff
@@ -37,7 +37,7 @@ if(merge||matching&&client.charcall.allData(client,userid,charid,"revived")){
   let sec = client.landMap.get(local[4],local[0]);
   let occList = sec[local[1]][local[2]][2][local[3]][4];
 
-let simpleMerge = ["port","cards","scards"];
+let simpleMerge = ["port","cards","scards", "b"];
```

Comment:
Heck, I didn't mean to include this as part of this commit. This code still hasn't even been tested.